languages/nim/exercises/concept/basics/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "ynfle",
+      "exercism_username": "ynfle"
+    }
+  ],
+  "forked_from": ["javascript/basics", "python/basics"]
```

Comment:
I assume this is how two tracks are credited

Reply:
Yep!